### DISCUSSION TITLE
Flux check --pre requires authentication

### DISCUSSION THREAD

**Discussion by jedi0606** [fluxcd/flux2#4004] [Q&A]
When I run "flux check --pre 2>&1" it asks me to log in to Azure. But I run it from the Gitlab pipeline, so it's not comfortable to log in during the pipeline run. Is there a way to disable this authentication?

**Comment by stefanprodan** [comment]
Flux check verifies your cluster version, so it needs access, why are you running it in a pipeline?

**Reply by jedi0606** [reply]
We try to have everything automated, including Flux bootstrapping. Firstly we run "flux check --pre 2>&1" and then "flux bootstrap git ....", it works, but we have to use the device code from the pipeline to log in to Azure manually, which means that the setup is not fully automatic. So, I'm trying to find out if it's possible to get rid of any manual steps.
